src/Controller/TiqrAppApiController.php

```diff
@@ -39,32 +42,22 @@
  */
 class TiqrAppApiController extends AbstractController
 {
-    private $tiqrService;
-    private $userRepository;
-    private $logger;
-    private $authenticationRateLimitService;
-
     public function __construct(
-        TiqrServiceInterface $tiqrService,
-        TiqrUserRepositoryInterface $userRepository,
-        AuthenticationRateLimitServiceInterface $authenticationRateLimitService,
-        LoggerInterface $logger
+        private readonly TiqrServiceInterface $tiqrService,
+        private readonly TiqrUserRepositoryInterface $userRepository,
+        private readonly AuthenticationRateLimitServiceInterface $authenticationRateLimitService,
+        private readonly LoggerInterface $logger
     ) {
-        $this->tiqrService = $tiqrService;
-        $this->userRepository = $userRepository;
-        $this->logger = $logger;
-        $this->authenticationRateLimitService = $authenticationRateLimitService;
     }
 
     /**
      * Metadata endpoint.
      *
      * The endpoint where the Tiqr app gets it's registration information from during enrollment
-     *
-     * @Route("/tiqr.php", name="app_identity_registration_metadata", methods={"GET"})
-     * @Route("/tiqr/tiqr.php", methods={"GET"})
      */
-    public function metadataAction(Request $request)
+    #[Route(path: '/tiqr.php', name: 'app_identity_registration_metadata', methods: ['GET'])]
+    #[Route(path: '/tiqr/tiqr.php', methods: ['GET'])]
```

Comment:
Two routes for the same contoller method? With explicit extension? Just wondering why?

Reply:
Yup thats a cool feature, you can reuse actions for multiple routes. In this case the tiqr server calls back to the GSSP on these two different endpoints. Could be fixed in the Tiqr server. But for the time being, leaving this a like this.